dokka-runners/runner-gradle-plugin-classic/src/main/kotlin/org/jetbrains/dokka/gradle/kotlin/kotlinCompilationUtils.kt

```diff
@@ -6,14 +6,12 @@ package org.jetbrains.dokka.gradle.kotlin
 
 import org.gradle.api.Project
 import org.jetbrains.dokka.gradle.kotlin
-import org.jetbrains.kotlin.gradle.dsl.KotlinCommonOptions
 import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
 import org.jetbrains.kotlin.gradle.dsl.KotlinSingleTargetExtension
 import org.jetbrains.kotlin.gradle.plugin.KotlinSourceSet
 import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinCommonCompilation
 
-internal typealias KotlinCompilation =
-        org.jetbrains.kotlin.gradle.plugin.KotlinCompilation<KotlinCommonOptions>
+internal typealias KotlinCompilation = org.jetbrains.kotlin.gradle.plugin.KotlinCompilation<*>
```

Comment:
`KotlinCommonOptions` is deprecated

Reply:
I'm surprised that it worked :) I thought we would need to use the replacements here, which could lead to compatibility problems with the older versions of KGP